### ISSUE TITLE
Is this project active?

### ISSUE THREAD

**Issue by sanjeevtripurari** [yahoo/kafka-manager#350]
Hi,

Not many queries are been answered or it looks any active development is happening.
Is this project active on GitHub or any where else?

**Comment by yctn** [comment]
alot of people wanne use this but it seems they dont maintain it anymore. i suggest if they dont maintain it anymore then fork it and lets continue on another repo.

**Comment by BDeus** [comment]
+1 on this.
With kafka 0.10.2, a lot of new admin feature will be available like real api with policy or new jmx metrics by partitions.
I think a fork can be very helpful. I actually use the fork of @Innometrics

**Comment by sanjeevtripurari** [comment]
+1 to support if we have active members to continue this project, either this or any other forked version.

**Comment by patelh** [comment]
Would be interested to hear what people think about moving this to an Apache project.

**Comment by simplesteph** [comment]
@patelh yes please. I think there's a lot of goodwill to improve this project

**Comment by yctn** [comment]
@patelh @simplesteph Agree. so far there is not a realy good/handy GUI tool to manage/monitor kafka would be great to improve this project

**Comment by sanjeevtripurari** [comment]
What is pre-requisite to move this project to Apache project, do we have PMC from Apache projects here or I can contact and get the update.

**Comment by aries-zhang** [comment]
@BDeus I can't find this project's fork in @Innometrics 's list, are you still using their fork?

**Comment by BDeus** [comment]
No i use the master now
the pull request has been merged since february https://github.com/yahoo/kafka-manager/pull/282

**Comment by patelh** [comment]
I would love to get this into Apache.
